Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -226,6 +226,8 @@
 static const BOOL defaultFastClickingEnabled = NO;
 #endif
 
+static void *readyForDisplayKVOContext = &readyForDisplayKVOContext;
```

Comment:
This name seems wrong? "Ready for display" is an AVKit KVO property; you should rename this after your own!